targets/l2_switch/p4src/l2_switch.p4

```diff
@@ -1,5 +1,5 @@
 //uncomment to enable openflow
-#define OPENFLOW_ENABLE
+//#define OPENFLOW_ENABLE
```

Comment:
This will break the openflow test I'm running on this target. Let me work on a more permanent solution to the mininet issue.